hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -511,12 +492,52 @@
       stage: V2
       tests: []
     }
+    {
+      name: enable
+      desc: '''
+            Verify that DUT connects to the USB when enable is set.
+
+            - set USB_CTRL_enable pin high.
+            - Make sure 'phy_config.pinflip' isn't set.
+            - Verify that the 'usb_dp_pullup_o' pin goes high.
+            '''
+      stage: V2
+      tests: ["usbdev_enable"]
+    }
+    {
+      name: resume_link_active
+      desc: '''
+            Verify that when Write a 1 to this bit to instruct usbdev to jump to the LinkResuming state
+            then write will only have an effect when the device is in the LinkPowered state.
+
+            - Perform no operation in J state for more than 3 ms.
```

Comment:
Grammar nit: I'd suggest rephrasing this as something like "Sit in J state for more than 3ms, performing no operation." (otherwise it looks like we need to make sure that we're ensuring none of the operations we perform takes more than 3ms :-D)